src/lib/seam/components/DeviceDetails/DeviceDetails.stories.tsx

```diff
@@ -18,6 +18,7 @@ const meta: Meta<typeof DeviceDetails> = {
       type: 'figma',
       url: 'https://www.figma.com/file/Su3VO6yupz4yxe88fv0Uqa/Seam-Components?type=design&node-id=358-39439&mode=design&t=4OQwfRB8Mw8kT1rw-4',
     },
+    layout: 'fullscreen',
```

Comment:
Storybook adds a default padding to previews, this removes it.

I wonder how many other renders aren't perfectly accurate due to this...